clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ClioDocument.scala

```diff
@@ -23,6 +25,13 @@ trait ClioDocument {
     * during recovery without needing to use a document mapper.
     */
   val entityId: String
+
+  /**
+    * The filename used for persisting a document's upsert data.
+    *
+    * @return the filename where this document's upsert data is stored
+    */
+  def persistenceFilename: String = ClioDocument.persistenceFilename(upsertId)
```

Comment:
I don't think we need this, since it just forwards to the method on the companion object (which is visible everywhere).

Reply:
If I remove it, how would `writePath.resolve(s"${document.persistenceFilename}")` work?

Reply:
`writePath.resolve(s"${ClioDocument.persistenceFilename(document.upsertId)}")`